src/calculateRentalCost.js

```diff
@@ -5,6 +5,18 @@
  */
 function calculateRentalCost(days) {
   // write code here
+  const cost = 40;
```

Comment:
The variable 'cost' should be defined as a constant outside the function and given a descriptive name to avoid magic numbers. Consider renaming it to 'DAILY_RENTAL_RATE' and using uppercase to indicate it is a constant.